<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Android_emulator-5556_1.2_1"
           tests="3"
           failures="2"
           errors="0"
           skipped="1"
           time="101.945">
  <properties />
            <testcase name="results" time="0.002" classname="AutomationSuites.nativeApp.P0_TestCases">
                <failure type="java.lang.Error"
                  message="Unresolved compilation problem: 
	The method testResult(AppiumDriver&lt;?&gt;, ITestResult) in the type mobDriverConfig is not applicable for the arguments (ITestResult)
"
              >
<![CDATA[java.lang.Error: Unresolved compilation problem: 
	The method testResult(AppiumDriver<?>, ITestResult) in the type mobDriverConfig is not applicable for the arguments (ITestResult)

  AutomationSuites.nativeApp.P0_TestCases.results(P0_TestCases.java:218)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:634)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
  org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
  java.util.concurrent.FutureTask.run(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="addLead" time="101.943" classname="AutomationSuites.nativeApp.P0_TestCases">
                <failure type="java.lang.AssertionError"
                  message="expected [AutoHL_GaTdt,] but found [AutoHL_GaTdt, ]"
              >
<![CDATA[java.lang.AssertionError: expected [AutoHL_GaTdt,] but found [AutoHL_GaTdt, ]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertEqualsImpl(Assert.java:137)
  org.testng.Assert.assertEquals(Assert.java:118)
  org.testng.Assert.assertEquals(Assert.java:453)
  org.testng.Assert.assertEquals(Assert.java:463)
  frameworkLibrary.mobDriverConfig.assertEquals_Text(mobDriverConfig.java:340)
  nativeApp.LeadsPage.verifyData_LeadsPage(LeadsPage.java:38)
  AutomationSuites.nativeApp.P0_TestCases.addLead(P0_TestCases.java:42)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:571)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
  org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
  java.util.concurrent.FutureTask.run(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
  
            <testcase name="Quit" time="0.0" classname="AutomationSuites.nativeApp.P0_TestCases">
          <skipped />
    </testcase>
  
  
</testsuite>
